docs/python/conceptual/query-engine/parallelization.md

```diff
@@ -3,47 +3,32 @@ title: Parallelizing queries
 sidebar_label: Parallelization
 ---
 
-Deephaven supports using multiple processors to speed up query evaluation. The extent to which Deephaven employs
-multiple processors depends on both the phase of operation and the query itself.
+Deephaven supports using multiple processors to speed up query evaluation. The extent to which Deephaven employs multiple processors depends on both the phase of operation and the query itself.
 
 ## Query initialization and updates
 
 When considering Deephaven query performance, there are two distinct phases to consider: initialization and updates.
 
 ### Query initialization
```

Comment:
I find both the query initialization and query updates sections to be unenlightening.  I doubt a reader is going to get the key ideas they need to walk away with.  There is extraneous stuff that distracts from the ideas, and the key ideas are not covered well.

1. Query initialization: When a table is first created, the initial state is computed using the available data.
2. Query updates: After initialization, a table is updated as input data changes.

I don't know that those are the right words, but those are the concepts.  I don't know that more words make the ideas more clear or provide more value.  This is so short that these subsections could go away, and these bullets could just go under `query phases`.